sphinx/source/docs/user_guide/jupyter.rst

```diff
@@ -129,96 +129,92 @@ to the URL of your JupyterHub installation. By default, JupyterHub will set
         full_url = urllib.parse.urljoin(user_url, proxy_url_path)
         return full_url
 
-Finally, you can pass the function you defined in step 2 to |show|
-as the notebook_url keyword argument, which Bokeh will call while
-setting up the server and creating the URL for loading the graph:
+3. Pass the function you defined in step 2 to the |show| function
+   as the notebook_url keyword argument, which Bokeh will call while
```

Comment:
The ``notebook_url`` should be marked as inline code (backquotes)

Reply:
Also, while you're at it: I believe this sentence might be easier to read if it gets broken up into two sentences. For example:

> ... as the ``notebook_url`` keyword argument. Bokeh then calls this function when it sets up the server and creates the URL to load a graph:

Reply:
Also just to remind (since it has tripped me up numerous times) it has to be *double* backquotes in sphinx

Reply:
Yeah, sent me googling too :-)